### ISSUE TITLE
Removing debugging output from 'hence machine init'

### ISSUE THREAD

**Issue by homertherefore** [hence-io/cli#7]
When your machine starts to build after finishing the prompts, some debugging data is being output to the console:

``` bash
? Everything is set, shall we proceed to install your VM now? Yes
--------------------------------------------------------------------------------
[16:25:18]  Cloning into '.'...

answers { dirs: { template: { customConfig: '/usr/local/lib/node_modules/hence-cli/lib/machine/init/template/custom.rb' } },
  files: [ '/usr/local/lib/node_modules/hence-cli/lib/machine/init/template/custom.rb' ],
  aborted: false,
  skipToInstall: false,
  installOption: 'Set up my own custom configuration.',
  dest: '/Users/Sean/henceTest',
  name: 'henseTest',
  ip: '172.19.8.101',
  port: 8080,
  cpus: 2,
  memory: 2048,
  customConfig: '$vm_name = "henseTest"\n$private_ip_prefix = "172.19.8"\n$private_ip_end = 101',
  beginInstall: true }
```

**Comment by ghost** [comment]
Should be fixed in v0.0.12.  Please try again and close on confirmation.

**Comment by homertherefore** [comment]
All clear!
